packages/components/src/Columns/Columns.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Children } from 'react';
+import { useStyles } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-keyed-flatten-children';
+
+type ColumnsProps = {
+  className?: string;
+  space?: number;
+  horizontalAlign?: 'left' | 'right' | 'center';
+  verticalAlign?: 'top' | 'bottom' | 'center';
+  title?: string; // Should only be used for testing.
+};
+
+export const Columns: React.FC<ColumnsProps> = ({
+  space = 0,
+  horizontalAlign = 'left',
+  verticalAlign = 'top',
+  className,
+  children,
+  ...props
+}) => {
+  let columnItems = flattenChildren(children);
+  let childClassNames = useStyles({ p: `${space / 2}px` });
+
+  // horizontal Alignment
+  let justify = 'flex-start';
+  if (horizontalAlign === 'right') {
+    justify = 'flex-end';
+  } else if (horizontalAlign === 'center') {
+    justify = 'center';
+  }
+
+  // vertical Alignment
+  let alignItems = 'flex-start';
+  if (verticalAlign === 'bottom') {
+    alignItems = 'flex-end';
+  } else if (verticalAlign === 'center') {
+    alignItems = 'center';
+  }
+
+  return (
+    <Box p={space} display="flex" className={className}>
+      <Box
+        width={`calc(100% + ${space}px)`}
```

Comment:
Can you help me understand what this does? Why do we need this? This does not work, right? since `space` is not necessarily equal to the pixel, right? The other box might have a different pixel value, depending on the theme.

Reply:
It works. This calculation is to set padding between different columns and include them to 100%. That you can do something like in the first example. Three Columns width: 4/12 + padding and you get 100%. At the moment the number you give to space is px. If we take the space values from the theme then this would be changed a little bit. It's just a proposal from me.

Reply:
Yeah, sorry. you' re right. That so confusing :D but you're right!